src/components/new-layout/QLayoutDrawer.js

```diff
@@ -133,6 +129,9 @@ export default {
     }
   },
   computed: {
+    belowBreakpoint () {
+      return this.view === 'mobile' || (this.view !== 'desktop' && this.breakpoint >= this.layout.width)
+    },
```

Comment:
belowBreakpoint is not a computed property on purpose to avoid re-renders when layout width changes. This should be avoided.

Reply:
Fixed. Sorry, didn't realize it.